app/services/slack_service.rb

```diff
@@ -0,0 +1,183 @@
+module SlackService
+  class InvalidRequest < RuntimeError;
+  end
+  class InvalidCommand < RuntimeError;
+  end
+
+  public
+
+  def create_post(command_text, team_id, user_id)
+    slack_users = command_text.scan(/(?=<).*?(?<=>)/)
+
+    receivers = []
+
+    slack_users.each do |slack_user|
+      id = slack_user[slack_user.index('@') + 1..slack_user.index('|') - 1]
+      user = User.where(slack_id: id).take
```

Comment:
Shouldn't this be scoped to a Kudo team?

Reply:
We could do that but the Post class has a build in check to make sure each receiver is a team member.

https://github.com/kabisa/kudo-o-matic/blob/7c6e82f08751a51d4b6097b79320486e736477fd/app/models/post.rb#L71-L77